### PR TITLE
add figure from the MLM demo paper

### PR DESCRIPTION
**Pull request by rbavery**
## Description

This adds the figure from the demo paper to the README
## Type of Change

- [x] :books: Examples, docs, tutorials or dependencies update;
- [ ] :wrench: Bug fix (non-breaking change which fixes an issue);
- [ ] :clinking_glasses: Improvement (non-breaking change which improves an existing feature);
- [ ] :rocket: New feature (non-breaking change which adds functionality);
- [ ] :boom: Breaking change (fix or feature that would cause existing functionality to change);
- [ ] :closed_lock_with_key: Security fix.

## Checklist

- [x] I've read the [`CONTRIBUTING.md`](../CONTRIBUTING.md) guide;
- [x] I've updated the code style using `make check`;
- [ ] I've written tests for all new methods and classes that I created;
- [ ] I've written the docstring in `Google` format for all the methods and classes that I used.

### CODE REVIEW

File: STAC_MLM.png
Comment (fmigneault) [inline STAC_MLM.png:1]: Move the image under `docs/static`.

File: README.md [hunk 1]
```diff
@@ -51,6 +51,8 @@ The main objectives of the extension are:
 2) record all necessary bands, parameters, modeling artifact locations, and high-level processing steps to deploy
    an inference service.
 
+<img src="./STAC_MLM.png" width="1000px" alt="STAC_MLM">
```
Comment (fmigneault) [inline README.md:54]: Do not enforce some width. Doesn't work nicely with other devices that your standard monitor. Simply use a markdown image reference and let it auto-scale. 

Also, I think it makes more sense to place it just before the "See Best Practices..."  below. Here, it cuts the summary description in two. 

Consider adding the citation as well under it. Just another way to communicate the paper reference along #32. 
> Francis Charette-Migneault, Ryan Avery, Brian Pondi, Joses Omojola, Simone Vaccari, Parham Membari, Devis Peressutti, Jia Yu, and Jed Sundwall. 2024. Machine Learning Model Specification for Cataloging Spatio-Temporal Models (Demo Paper). In 3rd ACM SIGSPATIAL International Workshop on Searching and Mining Large Collections of Geospatial Data (GeoSearch’24), October 29–November 1 2024, Atlanta, GA, USA. ACM, New York, NY, USA, 4 pages. https://doi.org/10.1145/3681769.3698586

Comment (rbavery) [reply README.md]: thanks, done!

File: docs/static/stac_mlm.png
Comment (fmigneault) [inline docs/static/stac_mlm.png:1]: Can you remove the transparent border?
It causes the figure and citation to not align.

Comment (rbavery) [reply docs/static/stac_mlm.png:1]: done, I changed it to a solid background. Figma doesn;t have other options besides transparent, solid, and grid background.

**Review by fmigneault — APPROVED** [review]

File: docs/static/stac_mlm.png
Comment (fmigneault) [reply docs/static/stac_mlm.png:1]: I've updated it using the figure from the paper that already removed the border.

**Pull request merged**